diff -Naur /tmp/discount-1.4.4/main.c ./main.c
--- /tmp/discount-1.4.4/main.c	Thu Jul  9 06:11:34 2009
+++ ./main.c	Wed Jul  8 22:22:13 2009
@@ -146,9 +146,10 @@
 			exit(1);
 		    }
 		    break;
-	default:    fprintf(stderr, "usage: %s [-dV] [-burl-base]"
-				    " [-F flags] [-f{+-}setting]"
-				    " [-o file] [file]\n", pgm);
+	default:    fprintf(stderr, "usage: %s [-dTV] [-b url-base]"
+				    " [-F bitmap] [-f {+-}flags]"
+				    " [-o ofile] [-s text]"
+				    " [-t text] [file]\n", pgm);
 		    exit(1);
 	}
     }
diff -Naur /tmp/discount-1.4.4/markdown.1.in ./markdown.1.in
--- /tmp/discount-1.4.4/markdown.1.in	Thu Jul  9 06:11:34 2009
+++ ./markdown.1.in	Thu Jul  9 00:21:13 2009
@@ -9,9 +9,11 @@
 .Sh SYNOPSIS
 .Nm
 .Op Fl d
+.Op Fl T
 .Op Fl V
+.Op Fl b Ar url-base
 .Op Fl F Pa bitmap
-.Op Fl f Ar options
+.Op Fl f Ar flags
 .Op Fl o Pa file
 .Op Fl s Pa text
 .Op Fl t Pa text
@@ -28,10 +30,14 @@
 .Pp
 The options are as follows:
 .Bl -tag -width "-o file"
+.It Fl b Ar url-base
+Links in source begining with / will be prefixed with
+.Ar url-base
+in the output.
 .It Fl d
 Instead of writing the html file, dump a parse
 tree to stdout.
-.It Fl f options
+.It Fl f Ar flags
 Set or clear various translation flags.   The flags
 are in a comma-delimited list, with an optional
 .Ar +
